Joedmin-hw3/Joedmin-hw3/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace Joedmin_hw3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.Write("What do you want to do? login/register/exit: ");
+                string answer = Console.ReadLine();
+
+                // Exit
+                if (String.Equals(answer, "exit", StringComparison.OrdinalIgnoreCase))
+                {
+                    Process.GetCurrentProcess().Kill();
+                }
+
+                // Login
+                else if (String.Equals(answer, "login", StringComparison.OrdinalIgnoreCase))
+                {
+                    using (var stream = new StreamReader("Users.txt"))
+                    {
+                        Dictionary<string, string> usersAndPasswords = GetCredentials("Users.txt");
+
+                        string[] credentials = GetCredencials();
+
+                        if (usersAndPasswords.ContainsKey(credentials[0]))
+                        {
+                            if (!(string.Equals(credentials[1], usersAndPasswords[credentials[0]])))
+                                Console.WriteLine("Invalid password.");
```

Comment:
Here you choose to do a bracketless "1-line" `if` but then your `else` statement is 1 line and doesn't do the same.  Either do...

```
if (...)
{
  // ABC
}
else
{
  // XYZ
}
```

Or do...

```
if (...)
  // ABC
else
  // XYZ
```

Reply:
Oh I see, looks like I missed that one, sorry